2dcontext/drawing-images-to-the-canvas/2d.drawImage.zerocanvas.html

```diff
@@ -25,15 +25,12 @@
 var canvas2 = document.createElement('canvas');
 canvas2.width = 0;
 canvas2.height = 10;
-assert_throws("INVALID_STATE_ERR", function() { ctx.drawImage(canvas2, 0, 0); });
```

Comment:
Yes, that makes perfect sense. In the new commit, I change this line to be ctx.drawImage(canvas2, 0, 0);